README.md

```diff
@@ -1 +1,47 @@
-# Riyaz-Portfolio
\ No newline at end of file
+# Riyaz-Portfolio
+
```

Comment:
[OPTIONAL] I recommend you add a brief project description, it'll make your documentation look more professional.

Reply:
Not needed as such , it